src/Cli/dotnet/Program.cs

```diff
@@ -204,18 +221,9 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null
             }
             else if (BuiltInCommandsCatalog.Commands.TryGetValue(parseResult.RootSubCommandResult(), out var builtIn))
             {
```

Comment:
There is a removal of a call to `TelemetryEventEntry.SendFiltered` that used to occur for built-in commands.  Is this intentional?

Reply:
I think it is intentional, since we send the same information on line 213.

Reply:
Correct. In 5.0.3xx, we send the top level command, then parse results, then send sublevelcommands in specific situations: https://github.com/dotnet/sdk/blob/d5f7e4d4d723c3db549241f028397bc1242ea7de/src/Cli/dotnet/Program.cs#L265 (see lines 257, 265, and then 273).

You can also see this in the Filter method where the two conditions in 5.0.3xx get collapsed down to one that does it all: https://github.com/dotnet/sdk/blob/1182ead7b832d77b10cc08ec74d71e066f5b51bc/src/Cli/dotnet/Telemetry/TelemetryFilter.cs#L27

Reply:
Cool, just wanted to make sure.